src/git-versioning.ts

```
@@ -0,0 +1,562 @@
+  private cleanStaleLock(): void {
+    const lockPath = path.join(this.vaultPath, '.git', LOCK_DIR_NAME);
+    if (!fs.existsSync(lockPath)) return;
+
+    try {
+      const stat = fs.statSync(lockPath);
+      const age = Date.now() - stat.mtimeMs;
+      if (age > STALE_LOCK_MS) {
+        fs.rmdirSync(lockPath);
+        logToFile('WARN', 'Removed stale commit lock', { age: Math.round(age / 1000), lockPath });
+      }
+    } catch { }
```

ESLint

[error] Empty block statement.

(no-empty)